### PR TITLE
Add docs about how to use stateful functions in pulsar

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolongran <xiaolongran@tencent.com>

### Motivation

Add docs about how to use stateful functions in pulsar

### CODE REVIEW

**Review by zymap — APPROVED** [review]

**Comment by wolfstudy** [comment]
ping @Anonymitaet @Jennifer88huang PTAL the docs.

File: site2/docs/functions-worker.md [hunk 1]
```diff
@@ -42,6 +42,38 @@ If authentication is enabled on the BookKeeper cluster, configure the following
...
+Escape character is '^]'.
+```
+
+2. Turn on this function in `functions_worker.yml`, as follows:
+
+```text
+stateStorageServiceUrl: bk://<bk-service-url>:4181
+```
+
+- bk-service-url: the service url points to bookkeeper table service.
```
Comment (Anonymitaet) [inline site2/docs/functions-worker.md:75]:
```suggestion
If you want to use Stateful-Functions related functions (for example,  `putState()` and `queryState()` related interfaces), follow steps below.
		
1. Enable the **streamStorage** service in the BookKeeper. 
   
   Currently, the service uses the NAR package, so you need to set the configuration in `bookkeeper.conf`.
    
    ```text
    extraServerComponents=org.apache.bookkeeper.stream.server.StreamStorageLifecycleComponent
    ```
    
    After starting bookie, use the following methods to check whether the streamStorage service is started correctly.
    
    Input

    ```shell
    telnet localhost 4181
    ```

    Output

    ```text
    Trying 127.0.0.1...
    Connected to localhost.
    Escape character is '^]'.
    ```
    
2. Turn on this function in `functions_worker.yml`.
   
    
    ```text
    stateStorageServiceUrl: bk://<bk-service-url>:4181
    ```
    
    `bk-service-url` is the service URL pointing to the BookKeeper table service.
```

do you mean this?

**Review by Anonymitaet — COMMENTED** [review]
Many thanks for your contribution!

File: site2/docs/functions-worker.md (hunk 1 above)
Comment (wolfstudy) [reply site2/docs/functions-worker.md]: Cool

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**